### DISCUSSION TITLE
Install MOOSE using WSL windows

### DISCUSSION THREAD

**Discussion by miraboreasumr** [idaholab/moose#23125] [Q&A Getting Started] [answered]
Hello, I am confused a little bit by the section on getting started, so it is not recommended to use Windows subsystem Linux？
Can I follow this https://mooseframework.inl.gov/getting_started/installation/conda.html
on WSL?

**Comment by aladshaw3** [comment, marked as answer]
I have had success installing MOOSE on WSL. Once you follow these instructions first (https://mooseframework.inl.gov/getting_started/installation/windows10.html), which setup your Ubuntu install in WSL, you should just be able to follow the conda instructions you reference for the Linux install. 

For the WSL installation, I don't use Peacock, so I have never tried installing the VcXsrv tool set before. I generally just skip that step. 

It has been a few months since I have gone through this, but I do remember needing to also manually install the gcc toolchain for WSL. You may also want to do the following:

```
sudo apt-get install build-essential
```

That should get you gcc, g++, make, etc. If you are missing any tools, you should be able to just manually install anything using `sudo apt-get install [tool]` from this point on.

**Reply by miraboreasumr** [reply]
Hello, I got this issue
```
user:/mnt/d/MOOSE$ mamba activate moose

Run 'mamba init' to be able to run mamba activate/deactivate
and start a new shell session. Or use conda to activate/deactivate.
```
after I type mamba init, and mamba activate moose
```
Run 'mamba init' to be able to run mamba activate/deactivate
and start a new shell session. Or use conda to activate/deactivate.
```
this happens again

**Reply by GiudGiud** [reply]
Did you remember to start a new shell session?

**Reply by miraboreasumr** [reply]
> Did you remember to start a new shell session?

Thanks, another question, where do you install moose, I follow the instructions, it is ~/projects, so it is in the root right? What do you suggest, I want to do /mnt/d for the future APP, but it won't allow me

**Reply by GiudGiud** [reply]
you should be able to
If you cant it s likely that you do not have permission to write to this directory
